src/components/ReserveOverviewBox.tsx

```diff
@@ -27,9 +32,11 @@ export function ReserveOverviewBox({ title, children }: ReserveOverviewBoxProps)
           padding: '8px',
         }}
       >
-        <Typography variant="secondary14" color="text.secondary" component="span">
-          {title}
-        </Typography>
+        {title && (
+          <Typography variant="secondary14" color="text.secondary" component="span">
+            {title}
+          </Typography>
+        )}
         {children}
```

Comment:
These changes just allow the component to be used full width rather than a 1/3, and an optional title, as the debt ceiling box doesn't use one.